### PR TITLE
added user field "number" (fixes #2214)

### LINKED ISSUE #2214: New users field 'number'
Add a new users field "number" which should help to map the membership/delegate number of a participant.

Show the number in the participants overview table (e.g. in a second line below the name). I don't like a separate column because it should be useable also for events without users numbers.

### PR DESCRIPTION
**Pull request by matakuka**
"Number" is a field for delegates/membership numbers. It will be displayed (1) on the participants' list below the name, (2) in the participant's details as "membership/delegate number". The CSV import now also has a field for the membership number.

I am not sure about the "migrations/0002_user_number.py file"- is it necessary? Can it be merged with other migrations files?

### CODE REVIEW

**Comment by normanjaeckel** [comment]
Pull request #2216 has to be merged before. Then we need a rebase here because of possible merge conflicts. So I came back on this soon.

**Comment by normanjaeckel** [comment]
@matakuka Now you can rebase this branch on top of current master. Please remove `openslides/users/migrations/0002_user_number.py` and create a new migrations file via `python manage.py makemigrations`. This file should be something like `0003_....py`.

**Comment by matakuka** [comment]
@normanjaeckel done.

File: openslides/users/static/templates/users/user-detail.html [hunk 1]
```diff
@@ -34,6 +34,8 @@ <h2 translate>Participant</h2>
         <div ng-repeat="group in user.groups_id">
           {{ (groups | filter: {id: group})[0].name | translate }}
         </div>
+      <label translate>Membership/delegate number</label>
```
Comment (emanuelschuetze) [inline openslides/users/static/templates/users/user-detail.html]: Please use "Participant number" (German: "Teilnehmernummer").

File: openslides/users/static/templates/users/user-import.html [hunk 2]
```diff
@@ -68,7 +68,7 @@ <h3 translate>Select a CSV file
   <h4 translate>Please note:</h4>
   <ul>
       <li><translate>Required comma or semicolon separated values with these column header names in the first row</translate>:<br>
-          <code>title, first_name, last_name, structure_level, groups, comment, is_active, is_committee</code>
+          <code>title, first_name, last_name, structure_level, number, groups, comment, is_active, is_committee</code>
```
Comment (emanuelschuetze) [inline openslides/users/static/templates/users/user-import.html]: You have to add the new number field also in the csv import preview table - see below...

**Comment by emanuelschuetze** [comment]
Some remarks.

**Comment by matakuka** [comment]
I've corrected the remarks, but noted that the participant number does not appear on the two printouts (users/print | /usr/password/print) yet. So it is not ready.

Also, probably different issue/request: The projected "user page" currently only projects the user name and title, nothing else. As such, it is quite uninformative. Maybe the "about me" or a future photo could be included (or the posts the user has been elected for/ been candidate)

**Comment by matakuka** [comment]
@emanuelschuetze ready again

File: openslides/users/static/templates/users/user-import.html [hunk 3]
```diff
@@ -90,6 +90,7 @@ <h3 translate>Preview</h3>
           <th translate>First name
           <th translate>Last name
           <th translate>Structure level
+          <th translate>Participant number
```
Comment (emanuelschuetze) [inline openslides/users/static/templates/users/user-import.html]: The  td column  of number is still missing - you only added the th line.

File: openslides/users/pdf.py [hunk 4]
```diff
@@ -23,8 +23,8 @@ def users_to_pdf(pdf):
     """
     Create a list of all users as PDF.
     """
-    data = [['#', _('Title'), _('Last Name'), _('First Name'),
-             _('Structure level'), _('Group')]]
+    data = [['#', _('Participant number'), _('Title'), _('Last Name'),
```
Comment (emanuelschuetze) [inline openslides/users/pdf.py]: I added some improvements for users list pdf in a new pull request, see https://github.com/matakuka/OpenSlides/pull/1. Please merge it first.

**Comment by matakuka** [comment]
@emanuelschuetze fixed table and merged your improvements

**Comment by normanjaeckel** [comment]
Ok. Please add CHANGELOG entry and rebase everything to one (or two or three) commits.

**Comment by matakuka** [comment]
done. {I moved the Assignment in CHANGELOG to get alphabetical order, as I assume this is desired (as the others are in this order)]

**Comment by matakuka** [comment]
(typo). Done now @normanjaeckel

**Comment by emanuelschuetze** [comment]
Ok for me. Ready for merge.

**Pull request merged**